agent/wpthook/test_state.cc

```diff
@@ -671,6 +686,10 @@ DWORD TestState::ElapsedMsFromStart(LARGE_INTEGER end) const {
   return ElapsedMs(_start, end);
 }
 
+DWORD TestState::ElapsedMsFromTimeoutStart(LARGE_INTEGER end) const {
+	return ElapsedMs(_timeout_start_time, end);

```

Comment:
The usual tab size that I have seen in most Google code and in Pat's code style is actually 2. I'd suggest sticking with 2.  You can easily tweak that in the VS preferences.